package.json

```diff
@@ -186,6 +193,8 @@
     "prop-types": "^15.6.0",
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
+    "react-error-overlay": "^5.1.2",
+    "react-helmet": "^5.2.0",
     "react-modal": "^3.8.1",
     "react-pose": "^4.0.6",
     "react-router-dom": "^4.3.1",
```

Comment:
It seems like this PR removes all the references to `react-router-dom` - is this dependency still needed?

Reply:
Nope! I just removed it.